lib/rules/linkedin/no-unguarded-window.js

```diff
@@ -0,0 +1,25 @@
+const { isBrowser, isEnvironmentBrowser } = require('../../utils/linkedin/bpr');
+const { isInActions, isInteractive } = require('../../utils/interactive');
+
+const MESSAGE = 'Do not use unguarded window references. Please see the following guide for more infromation: http://github.com/chadhietala/ember-best-practices/files/guides/no-unguarded-window.md';
+
+function isWindowReference(node) {
+  return node.object.name === 'window';
+}
+
+module.exports = {
+  meta: {
+    message: MESSAGE
+  },
+  create(context) {
+    return {
```

Comment:
Can you capture the import bindings and do the check. We ideally don't want to create reserved worlds or members. Failing case would be:

```
import Ember from 'ember';
let trollSteve = {
  isBrowser() { return 'tacos' }
}

export default Ember.Component.extend({
  init() {
   this._super(...arguments);
   if (trollSteve.isBrowser()) {
    let allTheTacos = true;
   }
  }
})
```
Also this is a case we want to cover and is legal.
```
import Ember from 'ember';
import environment from 'ember-stdlib/utils/environment';
const { isBrowser } = environment 

export default Ember.Component.extend({
  init() {
   this._super(...arguments);
   if (isBrowser()) {
    let allTheTacos = true;
   }
  }
})
```